src/lib/api/api-polyfill.ts

```diff
@@ -33,9 +33,27 @@ async function fetchHandler(
       // we get around that by renaming the file ext to .bin
       // see https://github.com/facebook/react-native/issues/27099
       // -prf
-      const newPath = reqBody.replace(/\.jpe?g$/, '.bin')
-      await RNFS.moveFile(reqBody, newPath)
-      reqBody = newPath
```

Comment:
Even if we revert, this isn't a good move. From testing, some of these files could not be moved and would result in an error.